controller/space_iterations.go

```diff
@@ -54,10 +54,18 @@ func (c *SpaceIterationsController) Create(ctx *app.CreateSpaceIterationsContext
 	}
 
 	return application.Transactional(c.db, func(appl application.Application) error {
-		_, err = appl.Spaces().Load(ctx, spaceID)
+		s, err := appl.Spaces().Load(ctx, spaceID)
 		if err != nil {
 			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err.Error()))
 		}
+		if !uuid.Equal(*currentUser, s.OwnerId) {
+			log.Warn(ctx, map[string]interface{}{
+				"space_id":     spaceID,
+				"space_owner":  s.OwnerId,
+				"current_user": *currentUser,
+			}, "user is not the space owner")
+			return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError("user is not the space owner"))
```

Comment:
It's better to return `jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("user is not the space owner"))` instead. And don't forget to add `a.Response(d.Forbidden, JSONAPIErrors)` to the `create` action in the design package